data/appdata-xml.m4

```diff
@@ -36,7 +36,7 @@ AU_DEFUN([APPDATA_XML],
          AS_IF([test "x$enable_appdata_validate" = "xyes"],
                [AC_MSG_ERROR([AppData validation was requested but appstream-util was not found])])])
 
-  AC_SUBST([appdataxmldir], [${datadir}/appdata])
+  AC_SUBST([appdataxmldir], [${datadir}/metainfo])
```

Comment:
I don't think we're quite ready for this change in distros; at least in fedora i'd need to make some changes to the base packages.